lnwallet/channel.go

```diff
@@ -3092,6 +3279,89 @@ func (lc *LightningChannel) createCommitDiff(
 	}, nil
 }
 
+// getDanglingRemoteUpdates returns all remote log updates that we haven't
+// signed for yet ourselves.
+func (lc *LightningChannel) getDanglingRemoteUpdates() []channeldb.LogUpdate {
```

Comment:
maybe use the term `ackedRemoteUpdates` to indicate these are the updates we just acked by revoking the commitment.

Reply:
Yes, they are acked, but that doesn't express that we didn't yet sign for it. Dangling is intentionally vague to cover which ones are in. `ackedUnsignedRemoteUpdates`?